src/screens/Loading.js

```diff
@@ -23,22 +23,66 @@ import colors from '../colors';
 import { generateAccountId } from '../util/account';
 import {
 	loadAccounts,
+	loadIdentities,
 	loadToCAndPPConfirmation,
-	saveAccount
+	saveAccount,
+	saveIdentities
 } from '../util/db';
+import {
+	extractAddressFromAccountId,
+	isEthereumAccountId
+} from '../util/identitiesUtils';
 
 export default class Loading extends React.PureComponent {
 	async componentDidMount() {
 		const tocPP = await loadToCAndPPConfirmation();
 		const { navigate } = this.props.navigation;
 		if (!tocPP) {
 			this.migrateAccounts();
+			this.migrateIdentity();
 			navigate('TocAndPrivacyPolicy');
 		} else {
 			navigate('Welcome');
 		}
 	}
 
+	// TODO migrate identities only on internal test devices, remove them in v4.1
```

Comment:
Migration code only to refactor the schema on internal test devices. Which is now only installed with Test Flight, so in the next release version, we could remove these code.